docs/reference/System_variable.md

```diff
@@ -153,29 +153,31 @@ SELECT /*+ SET_VAR(query_timeout = 1) */ sleep(3);
 
 * forward_to_master
 
-  Used to set whether some commands will be forwarded to the leader FE node for execution. The default value is false, meaning no forwarding. There are multiple FE nodes in StarRocks, one of which is the leader node. Normally, users can connect to any FE node for full-featured operation. However, some information is only available from the leader FE node.
+  Used to specify whether some commands will be forwarded to the leader FE for execution. The default value is `false`, meaning no forwarding. There are multiple FE nodes in StarRocks, one of which is the leader FE. Normally, users can connect to any FE node for full-featured operation. However, some information is only available on the leader FE node.
 
-  For example, if the `SHOW BACKENDS;` commend is not forwarded to the leader FE node, only basic information (e.g., whether the node is alive or not) can be seen. Forwarding to the leader FE can get more detailed information including the node start time, last heartbeat time, etc.
+  For example, if the SHOW BACKENDS command is not forwarded to the leader FE, only basic information (e.g., whether the node is alive or not) can be seen. Forwarding to the leader FE can get more detailed information including the node start time and last heartbeat time.
 
   The commands affected by this parameter are as follows:
 
-  * `SHOW FRONTENDS`: Forwarding to leader allows users to view the last heartbeat message.
+  * SHOW FRONTENDS: Forwarding to leader allows users to view the last heartbeat message.
 
-  * `SHOW BACKEND`: Forwarding to leader allows users to view the boot time, last heartbeat information, and disk capacity information.
+  * SHOW BACKENDS: Forwarding to leader allows users to view the boot time, last heartbeat information, and disk capacity information.
 
-  * `SHOW BROKER`: Forwarding to leader allows users to view the boot time and last heartbeat information.
+  * SHOW BROKER: Forwarding to leader allows users to view the boot time and last heartbeat information.
 
-  * `SHOW TABLET`;
-  * `ADMIN SHOW REPLICA DISTRIBUTION`;
-  * `ADMIN SHOW REPLICA STATUS`: Forwarding to leader allows users to view the tablet information stored in the leader FE metadata. Normally, the tablet information in the different FE metadata should be the same. In case of a problem, you can use this method to compare the differences in metadata of the current FE and the leader FE.
+  * SHOW TABLET
 
-  * `Show PROC`: Forwarding to leader allows users to view the PROC information stored in the metadata. This is mainly used for metadata comparison.
+  * ADMIN SHOW REPLICA DISTRIBUTION
+
+  * ADMIN SHOW REPLICA STATUS: Forwarding to leader allows users to view the tablet information stored in the leader FE metadata. Normally, the tablet information in the different FE metadata should be the same. In case of a problem, you can use this method to compare the differences in metadata of the current FE and the leader FE.
```

Comment:
FYR: Normally, the metadata regarding tablet information in the different FE should be the same.